benchmarks/bigint_conversion.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+import arkouda as ak
+import time
+
+
+def time_bigint_conversion(N, trials, seed, max_bits):
+    print(">>> arkouda uint arrays from bigint array")
+    cfg = ak.get_config()
+    print("numLocales = {}, N = {:,}".format(cfg["numLocales"], N))
+
+    a = ak.randint(0, 2**32, args.size, dtype=ak.uint64, seed=seed)
+    b = ak.randint(0, 2**32, args.size, dtype=ak.uint64, seed=seed)
+
+    convert_to_bigint_times = []
+    for i in range(trials):
+        start = time.time()
+        ba = ak.bigint_from_uint_arrays([a, b], max_bits=max_bits)
+        end = time.time()
+        convert_to_bigint_times.append(end - start)
+    avg_conversion = sum(convert_to_bigint_times) / trials
+
+    print("bigint_from_uint_arrays Average time = {:.4f} sec".format(avg_conversion))
+    print(
+        "bigint_from_uint_arrays Average rate = {:.4f} GiB/sec".format(
+            (2 * a.size * a.itemsize) / 2**30 / avg_conversion
+        )
+    )
+    print()
+    print(">>> arkouda bigint array to uint arrays")
+
+    convert_from_bigint_times = []
+    for i in range(trials):
+        start = time.time()
+        u_arrays = ba.bigint_to_uint_arrays()
+        end = time.time()
+        convert_from_bigint_times.append(end - start)
+    avg_conversion = sum(convert_from_bigint_times) / trials
+
+    print("bigint_to_uint_arrays Average time = {:.4f} sec".format(avg_conversion))
+    print(
+        "bigint_to_uint_arrays Average rate = {:.4f} GiB/sec".format(
+            (2 * a.size * a.itemsize) / 2**30 / avg_conversion
+        )
+    )
+    assert ak.all(a == u_arrays[0])
+    assert ak.all(b == u_arrays[1])
+
+
+def check_correctness(seed, max_bits):
+    N = 10**4
+
+    a = ak.randint(0, N, N, dtype=ak.uint64, seed=seed)
+    b = ak.randint(0, N, N, dtype=ak.uint64, seed=seed)
+    u_arrays = ak.bigint_from_uint_arrays([a, b], max_bits=max_bits).bigint_to_uint_arrays()
+
+    assert ak.all(a == u_arrays[0])
+    assert ak.all(b == u_arrays[1])
+
+
+def create_parser():
+    parser = argparse.ArgumentParser(
+        description="Measure the performance of converting between bigint and uint arrays."
+    )
+    parser.add_argument("hostname", help="Hostname of arkouda server")
+    parser.add_argument("port", type=int, help="Port of arkouda server")
+    parser.add_argument("-n", "--size", type=int, default=10**8, help="Problem size: length of array")
+    parser.add_argument("--max-bits", type=int, default=-1)
```

Comment:
Maybe add `help` since it isn't super obvious what it does (or that `-1` is a sentinel)?